verilog/analysis/verilog_linter.cc

```diff
@@ -210,6 +211,26 @@ LinterConfiguration LinterConfigurationFromFlags() {
   const auto& ruleset = absl::GetFlag(FLAGS_ruleset);
   config.UseRuleSet(ruleset);
 
+  // Read local configuration file
+  std::ifstream local_config(".rules.verible_lint");
```

Comment:
There is `verible::file::GetContents()`, which might be a utility function to use here for less code https://github.com/google/verible/blob/master/common/util/file_util.h

Reply:
Right, that makes perfect sense, thanks!